dataset_configs/armenian/text_mcv/config.yaml

```diff
@@ -0,0 +1,141 @@
+    output_manifest_file: ${workspace_dir}/manifest0.json
+
+  - _target_: sdp.processors.ReadTxtLines
+    input_file_key: source_filepath
+    text_key: text_line
+    output_manifest_file: ${workspace_dir}/manifest1.json
+
+  - _target_: sdp.processors.SubRegex
+    output_manifest_file: ${workspace_dir}/manifest2.json
+    text_key: text_line
+    regex_params_list:
+      - {"pattern": '։', "repl": ':'}
+      - {"pattern": '․', "repl": "."}
+      - {"pattern": '—', "repl": "-"}
+      - {"pattern": '–', "repl": "-"}
+      - {"pattern": '―', "repl": "-"}
+      - {"pattern": '\.\.\.', "repl": "…"}
+      - {"pattern": "\\s+", "repl": " "}
+
+  - _target_: sdp.processors.DuplicateFields
+    output_manifest_file: ${workspace_dir}/manifest3.json
+    duplicate_fields: {"text_line":"text"}
+
+  - _target_: sdp.processors.SplitLineBySentence
+    text_key: text
+    end_pattern: ':|\.|…'
+    output_manifest_file: ${workspace_dir}/manifest4.json
+
+  - _target_: sdp.processors.DropIfRegexMatch
+    output_manifest_file: ${workspace_dir}/manifest5.json
+    regex_patterns:
+      - 'տիկ\. $'
+      - 'Գ\. $'
+      - '[0-9]'
+      - '\('
+      - '\)'
+      - '\['
+      - '\]'
+      - '\*'
+      - '"'
+      - '«'
+      - '»'
+      - '[А-Яа-я]'
+      - '[A-Za-z]'
+      - '\+'
+      - '='
+      - '¬'
+      - '&'
+
+  - _target_: sdp.processors.DropNonAlphabet
+    output_manifest_file: ${workspace_dir}/manifest6.json
+    alphabet: "՝՞՜՛`֊´’'՚-ԱԲԳԴԵԶԷԸԹԺԻԼԽԾԿՀՁՂՃՄՅՆՇՈՉՊՋՌՍՎՏՐՑՒՓՔՕՖՈՒԵ աբգդեզէըթժիլխծկհձղճմյնշոչպջռսվտրցւփքօֆուև,:\\.…;"
+    test_cases:
+      - {input: {text: "test тест Գրիմ եղբայրներ, անտառի տնակը, Ռուսերենից թարգմանեց, Ամալիյա Ուկասյանը."}, output: null}
+      - {input: {text: "Գրիմ եղբայրներ, անտառի տնակը, Ռուսերենից թարգմանեց, Ամալիյա Ուկասյանը."}, output: {text: "Գրիմ եղբայրներ, անտառի տնակը, Ռուսերենից թարգմանեց, Ամալիյա Ուկասյանը."}}
+
+  - _target_: sdp.processors.CountNumWords
+    output_manifest_file: ${workspace_dir}/manifest7.json
+    alphabet: "ԱԲԳԴԵԶԷԸԹԺԻԼԽԾԿՀՁՂՃՄՅՆՇՈՉՊՋՌՍՎՏՐՑՒՓՔՕՖՈՒԵ աբգդեզէըթժիլխծկհձղճմյնշոչպջռսվտրցւփքօֆուև"
+    text_key: text
+    num_words_key: num_words
+
+  - _target_: sdp.processors.PreserveByValue
+    output_manifest_file: ${workspace_dir}/manifest8.json
+    input_value_key: num_words
+    target_value: 15
+    operator: le
+
+  - _target_: sdp.processors.PreserveByValue
+    output_manifest_file: ${workspace_dir}/manifest9.json
+    input_value_key: num_words
+    target_value: 3
+    operator: ge
+
+  - _target_: sdp.processors.langs.armenian.GetSourceBookName
+    output_manifest_file: ${workspace_dir}/manifest10.json
+    source_file_key: source_filepath
+    source_key: Source
+
+  - _target_: sdp.processors.RenameFields
+    output_manifest_file: ${workspace_dir}/manifest11.json
+    rename_fields: {"text": "Sentence"}
+    
+  - _target_: sdp.processors.KeepOnlySpecifiedFields
+    output_manifest_file: ${workspace_dir}/final_manifest.json
```

Comment:
should be `output_manifest_file: ${final_manifest}` since we specify `final_manifest` at the top of the file?

Reply:
yes, fixed